src/app/services/raiden.service.ts

```diff
@@ -813,6 +813,30 @@ export class RaidenService {
         );
     }
 
+    public setUdcDeposit(totalDeposit: BigNumber): Observable<void> {
```

Comment:
I'm not sure if `set` is the best verb here. It sounds way less impact-full than what this actually does according to the API documentation. It sounds a bit like getter/setter stuff from OOP. Maybe rather `depositToUdc` or something like this? But maybe the PC works differently there.

Reply:
Makes sense to me. I'll rename it. Thanks!